### PR TITLE
Divyanshu | Test| Windows (Firefox) - (1280*800) 

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/85ztzftv0

### CODE REVIEW

File: apps/web-giddh/src/app/voucher/voucher.component.scss [hunk 1]
```diff
@@ -341,6 +341,7 @@ textarea:focus {
                 padding: 10px 8px 20px;
                 white-space: nowrap;
                 position: relative;
+                background: rgba(0,0,0,0);
```
Comment (nisha-walkover) [inline apps/web-giddh/src/app/voucher/voucher.component.scss]: add variable

File: apps/web-giddh/src/assets/css/style-1.scss [hunk 2]
```diff
@@ -2841,6 +2849,10 @@ td.mat-footer-cell:first-of-type {
     white-space: normal !important;
     word-break: inherit;
 }
+td.mat-mdc-cell:first-of-type {
+    word-wrap: anywhere;
+}
+td
```
Comment (nisha-walkover) [inline apps/web-giddh/src/assets/css/style-1.scss]: format scss

**Review by nisha-walkover — APPROVED** [review]

File: apps/web-giddh/src/assets/magic-link-assets/magic.css [hunk 3]
```diff
@@ -1017,6 +1018,8 @@ body {
     float: left;
     width: 100%;
     margin-bottom: 0;
+    border-radius: .25rem 0 0 .25rem !important;
+    border: 1px solid #ced4da !important;
```
Comment (nisha-walkover) [inline apps/web-giddh/src/assets/magic-link-assets/magic.css]: add color variable

File: apps/web-giddh/src/assets/magic-link-assets/magic.css [hunk 4]
```diff
@@ -1043,6 +1046,10 @@ body {
     white-space: nowrap;
     vertical-align: middle;
 }
+.tooltip-inner{
+    background-color: #000 !important;
+    color: #fff !important;
```
Comment (nisha-walkover) [inline apps/web-giddh/src/assets/magic-link-assets/magic.css]: add color variable

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**